src/components/Modals/BuyItemSlotsModal/BuyItemSlotsModal.container.ts

```diff
@@ -22,6 +22,7 @@ const mapState = (state: RootState): MapStateProps => {
 const mapDispatch = (dispatch: MapDispatch): MapDispatchProps => ({
   onFetchThirdPartyItemSlots: () => dispatch(fetchThirdPartyItemTiersRequest()),
   onTierSelected: () => dispatch(clearTiersError()),
+  onBeforeClose: () => dispatch(clearTiersError()),
```

Comment:
This might be coincidental, but given that we have two methods with different names poiting to the same dispatched action, what if we just have `onClearTiersError()` and call that instead?

Reply:
I definitely though about it, but from the perspective of the component as an isolated chunk of code, there's nothing to explicitly clear the errors, so it might be a bit confusing.